common/templates/layouts/two-column.njk

```diff
@@ -0,0 +1,24 @@
+{% extends "./base.njk" %}
+
+{% set containerClass = ["govuk-grid-row"].concat(additionalContainerClass) | join(" ") %}
+
+{% block content %}
+
+  {% block contentHeader %}{% endblock %}
+
+  <div class="{{ containerClass }}">
+
+    <article class="govuk-grid-column-two-thirds">
```

Comment:
Haven't read up on this in a while but I wonder whether we actually need to use `<article>` and `<aside>` elements.

Having a poke around GOV.UK they haven't seemed to use them for article style pages. Don't know if its something that has phased out?

Reply:
So having a read over [html doctor](http://html5doctor.com/aside-revisited/) they are still relevant. I can change them to `<div>` if you like?

Reply:
This may also be due to support right? I see we aren't using `html5shiv` anywhere also for these elments

Reply:
Ah yes, maybe lets stick to divs for now then.